tests/test_encoding_stream_logprobs_http_honesty.py

```
@@ -3,23 +3,23 @@
 from pathlib import Path
-import sys
 
 sys.path.insert(0, str(Path(__file__).resolve().parents[1]))
 
-from contextual_orchestrator import ModelAgent, TaskOrchestrator  # noqa: E402
-from contextual_orchestrator.server import SecurityConfig, build_server  # noqa: E402
+from contextual_orchestrator import ModelAgent, TaskOrchestrator
+from contextual_orchestrator.server import SecurityConfig, build_server
 
-_TEST_AUTH_TOKEN = "encoding_stream_logprobs_http_honesty_token"  # noqa: S105
+_TEST_AUTH_TOKEN = "encoding_stream_logprobs_http_honesty_token"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "_TEST_AUTH_TOKEN"

(S105)